includes/data-stores/abstract-wc-order-item-type-data-store.php

```diff
@@ -59,11 +59,15 @@ public function create( &$item ) {
 	public function update( &$item ) {
 		global $wpdb;
 
-		$wpdb->update( $wpdb->prefix . 'woocommerce_order_items', array(
-			'order_item_name' => $item->get_name(),
-			'order_item_type' => $item->get_type(),
-			'order_id'        => $item->get_order_id(),
-		), array( 'order_item_id' => $item->get_id() ) );
+		$changes = $item->get_changes();
+
+		if ( ! empty( $changes ) ) {
```

Comment:
We have an array of changes, and we know the only things being updated here are `name` `type` and `order_id` - could this check be made more specific?